pkg/symbol/elfutils/debuginfofile.go

```diff
@@ -120,7 +143,11 @@ func (f *debugInfoFile) SourceLines(addr uint64) ([]profile.LocationLine, error)
 			name = getFunctionName(abstractOrigin)
 		}
 
-		file, line := findLineInfo(f.lineEntries[cu.Offset], ch.Ranges)
+		// An inlined subroutine entry may have these field:
+		// - DW_AT_call_line：line number of inlined subroutine call
+		// - DW_AT_call_file: File contain inlined subroutine call
+		file := f.lineFiles[cu.Offset][ch.Entry.Val(dwarf.AttrCallFile).(int64)].Name
```

Comment:
this relies on all dwarf infos to be correct and complete, but compilers are also software and sometimes make mistakes, so I would suggest we should check that each of these accesses are checked so we don't get a nil pointer panic